static/app/components/u2f/u2fContainer.tsx

```diff
@@ -45,11 +46,11 @@ class U2fContainer extends Component<Props, State> {
 
     return (
       <div className={className}>
-        {authenticators.map(auth =>
-          auth.id === 'u2f' && auth.challenge ? (
+        {authenticators.map(auth => {
+          return auth.id === 'u2f' && auth.challenge ? (
```

Comment:
Did something change here?

Reply:
i don't believe i made these changes, i think the auto-formatter might've done this

Reply:
reverted this to what it was before